services/src/main/groovy/org/jfrog/artifactory/client/impl/ArtifactoryImpl.java

```diff
@@ -28,329 +31,337 @@
-            }
-        }
-        if (content != null) {
-            httpPatch.setEntity(new StringEntity(content, contentType));
-        }
-        HttpResponse httpResponse = httpClient.execute(httpPatch);
-        if (object == String.class) {
-            return (T) Util.responseToString(httpResponse);
-        }
-
-        return Util.responseToObject(httpResponse, object, interfaceObject);
-    }
-
-    public <T> T put(String path, org.apache.http.entity.ContentType contentType, String content, Map<String, String> headers, InputStream inputStream, long length, Class<? extends T> object, Class<T> interfaceObject) throws IOException {
-        HttpPut httpPut = new HttpPut();
-        httpPut.setURI(URI.create(url + path));
-        httpPut = (HttpPut) addAccessTokenHeaderIfNeeded(httpPut);
-
-        if (contentType != null) {
-            httpPut.setHeader("Content-type", contentType.getMimeType());
-        } else {
-            contentType = org.apache.http.entity.ContentType.WILDCARD;
-        }
-
-        if (inputStream != null) {
-            InputStreamEntity entity = new InputStreamEntity(inputStream, length);
-            httpPut.setEntity(entity);
-        }
-
-        if (StringUtils.isNotBlank(content)) {
-            httpPut.setEntity(new StringEntity(content, contentType));
-        }
-        if (headers != null && !headers.isEmpty()) {
-            for (String key : headers.keySet()) {
-                // Need to do String value of since the value of the key can be non String element (for example boolean)
-                httpPut.setHeader(key, String.valueOf(headers.get(key)));
-            }
-        }
-        HttpResponse httpResponse = httpClient.execute(httpPut);
-        int status = httpResponse.getStatusLine().getStatusCode();
-        if (status == HttpStatus.SC_OK || status == HttpStatus.SC_NO_CONTENT || status == HttpStatus.SC_ACCEPTED || status == HttpStatus.SC_CREATED) {
-            if (object == String.class) {
-                return (T) Util.responseToString(httpResponse);
-            }
-            return Util.responseToObject(httpResponse, object, interfaceObject);
-        }
-
-        throw newHttpResponseException(httpResponse);
-    }
-
-    public String delete(String path) throws IOException {
-        HttpDelete httpDelete = new HttpDelete();
-
-        httpDelete.setURI(URI.create(url + path));
-        httpDelete = (HttpDelete) addAccessTokenHeaderIfNeeded(httpDelete);
-        HttpResponse httpResponse = httpClient.execute(httpDelete);
-        int status = httpResponse.getStatusLine().getStatusCode();
-        if (status != HttpStatus.SC_OK && status != HttpStatus.SC_NO_CONTENT && status != HttpStatus.SC_ACCEPTED) {
-            throw newHttpResponseException(httpResponse);
-        }
-        return Util.responseToString(httpResponse);
-    }
+	private static final String ORIGINAL_HOST_CONTEXT_PARAM = "original.host.context.param";
```

Comment:
Could it be that you used some kind a formatting tool on the whole file? This highly not recommended when you're contributing code, because it makes it hard to review the code. In addition, you're adding unnecessary changes to the code, which affects the history for whoever tries to read the code in the future.

Reply:
Totally Agree. But not sure why it didn't show up in my diff tool. Probably I need to switch to a better UI version of git diff tooling.